keras/preprocessing/image.py

```diff
@@ -330,7 +347,10 @@ def load_img(path, grayscale=False, target_size=None):
     if target_size:
         hw_tuple = (target_size[1], target_size[0])
         if img.size != hw_tuple:
-            img = img.resize(hw_tuple)
+            if interpolation not in _PIL_INTERPOLATION_METHODS:
+                raise ValueError('Invalid interpolation method specified')
```

Comment:
The ValueError should be listed in the docstring. The error message should specify what was passed, and the list of values expected instead.